apps/api/v1/serializers.py

```diff
@@ -65,11 +71,11 @@ class Meta:
 
 
 class UserSerializer(serializers.ModelSerializer):
-    full_name = serializers.SerializerMethodField()
+    title = serializers.SerializerMethodField()
 
     class Meta:
         model = User
-        fields = ('id', 'full_name',)
+        fields = ('id', 'title',)
 
-    def get_full_name(self, obj):
+    def get_title(self, obj):
```

Comment:
Why did you rename this to `get_title` it sounds like tries to get `Mr`, `Mrs`? When the Django originally called this method `get_full_name`

Reply:
So I could call title in react on the three search filters, i.e. `tags.title`, `organisation.title`, `user.title`

Reply:
Mostly for DRY purposes but could see the issue